README.md

```diff
@@ -1,8 +1,42 @@
+
+## Skills
+
+**Proficient:** Java, Javascript, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git
+
+**Competent:** Python, C, C++, React
+
+**Familiar:** Swift, Angular, Matlab, Haskell
+
+## Development Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_DaydeReid](https://github.com/jarviscanada/jarvis_data_eng_DaydeReid)
+
+- **[Cluster Monitor](./linux_sql)**: Implemented a series of bash scripts that record hardware specifications and usage statistics for a Linux cluster and write them to a PSQL database contained in a Docker instance for future analysis
+- **[Core Java Apps](./core_java)**: Developed three small Java apps: An app that replicates the recursive grep function found in Linux systems using Java streams, some basic DAO implementations for a JDBC system, and a Twitter CLI App, which utilizes a REST API to post, view, and delete tweets from the Twitter webpage 
+- **[SpringBoot App](./springboot)**: Not started
+- **[Cloud & DevOps](./cloud_devops)**: Not started
+- **[Hadoop](./hadoop)**: Not started
+- **[Spark/Scala](./spark)**:  Not started
+
+## Professional Experiences
+
+**Software Developer,  Jarvis, Toronto (2020-Present):** Implemented several projects, covering a wide variety of uses and technologies, including Linux cluster monitoring scripts and Java command line apps
+
+**Software Developer Co-op, Loblaw Digital, Toronto (2018-2018):** Colloborated with a team to ensure optimal working conditions of an existing React-based ecommerce site through the resolving of numerous technical issues and improvement of existing page designs
+
+**Application Developer Co-op, Parks Canada, Toronto (2017-2017):** Developed and released a guide application for Rouge National Urban Park on both iOS and Android platforms, which has since been downloaded by over 1000 users
+
+## Education & Academic Projects
+
+**University of Toronto Scarborough (2015-2019)**, Honours Bachelor of Science, Computer Science
+
+- **Lego Soccer Robot:** Designed an programmed an autonomous soccer robot, which used a C program to analyze web cam footage and inform a simple state-based AI's decision making
+
+- **Collaborative Coding Web App:** Designed and programmed a Python and Javascript-based web app allowing for the writing and running of Python code in browser, with multiple people able to edit the same code at the same time
```

Comment:
again, describe the tech stack. e.g. libraries, framework, design, architecture, platform, deployment